pkgs/by-name/pr/prometheus-folder-size-exporter/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  cargo,
+  rustc,
+  fetchFromGitHub,
+  rustPlatform,
+  lib,
+}:
+rustPlatform.buildRustPackage {
+  pname = "prometheus_folder_size_exporter";
+  version = "0.5.1";
```

Comment:
```suggestion
  version = "0.5.0-unstable-2023-02-22";
```

Reply:
I had forgotten why I had done this, thanks for showing me the proper way to reference something beyond the normal tag